### PR TITLE
styling on signup/login/navbar, favicon

### PR DESCRIPTION
**Pull request by guipace**
Styling and positioning of navbar, signup and login pages.
Small styling improvements to other areas.
Added a new favicon.

### CODE REVIEW

**Review by edherm — CHANGES_REQUESTED** [review]
Looking good! I've got just a few things for you to review before I approve this PR. Let me know when you've finished (which includes pulling main, merging that into your branch, solving merge conflicts, then pushing it back to this same branch), and I'll look it over once more.

Please do: In views/layout.pug lines 28-31: this is all repeated code! Please make a mixin to DRY this up. You just need to pass in the alt-text and github username.

Optional: This is a bigger change to all of your forms, but you should also create an input mixin to replace all of this repetitive code. If you have other issues you want to work out first, you may just want to raise an issue on the repo and put it in your kanban/project board so you can get to it eventually.

**Comment by guipace** [comment]
All suggested changes have been made in addition to changing the logo on the nav bar.
This branch is already updated to the current version of main.

File: views/layout.pug [hunk 1]
```diff
@@ -25,10 +25,10 @@ html
 
       div(class='connectWithUs')
         h3(class="connectWithUs-h3") Connect With Us
-        div: a(href="https://github.com/guipace"): img(src="/images/githubicon.png", alt="Guilherme Pace")
-        div: a(href="https://github.com/HeroRoku"): img(src="/images/githubicon.png", alt="Jevon Timmons")
-        div: a(href="https://github.com/lanejohns"): img(src="/images/githubicon.png", alt="Lane Johns")
-        div: a(href="https://github.com/MattMav21"): img(src="/images/githubicon.png", alt="Matthew Hasan")
+        div: a(href="https://github.com/guipace"): img(class="connectWithUs-github" src="/images/githubicon.png", alt="Guilherme Pace")
```
Comment (edherm) [inline views/layout.pug]: Please change this (and the following three github profile links) into mixins to DRY up this section.

**Review by edherm — CHANGES_REQUESTED** [review]
Please add a new mixin in utils.pug that builds the github div-links for you. Then those links would just look something like `+githubLink(memberName, githubHandle)` instead of those many repeated lines of code.

**Review by edherm — APPROVED** [review]
Nice work on these mixins!

**Pull request merged**